README.md

```diff
@@ -2,5 +2,5 @@
 
```

Comment:
Could you add some descriptions (e.g. one-line definition, how to install, how to test)?
A one-line description example: "A python version of ain-js [link to https://www.npmjs.com/package/@ainblockchain/ain-js]."
cc: @liayoo

Reply:
Added some descriptions. Please check!